src/fragments/Graph.react.js

```diff
@@ -154,7 +206,28 @@ class PlotlyGraph extends Component {
         clearExtendData();
     }
 
+    isResponsive(props) {
+        const {config, figure, responsive} = props;
+
+        if (type(responsive) === 'Boolean') {
+            return responsive;
+        }
```

Comment:
If responsive is a boolean, use as-is. Otherwise, determined auto responsiveness by looking at config/figure for the graph.